GitUI/Theming/Renderers/ComboBoxRenderer.cs

```diff
@@ -195,20 +195,11 @@ private static void RenderDownArrow(Context ctx, State.DropDown stateid, Rectang
                 new Point(x3, y1)
             };
 
-            Color arrowColor;
-            switch (stateid)
+            Color arrowColor = stateid switch
             {
-                case State.DropDown.CBXS_DISABLED:
-                    arrowColor = SystemColors.ControlDark;
-                    break;
-
-                // case State.DropDown.CBXS_NORMAL:
```

Comment:
keep the comments?

Reply:
I omitted these intentionally, but can bring them back if there's a strong desire.

There are many other switches on enum values that don't include comments for missing members. For example, in `RenderTextEx`, plenty of the `Parts` enum members are omitted from code and comments.

Reply:
I omitted these intentionally. 

There are many other switches on enum values that don't include comments for missing members. For example, in `RenderTextEx`, plenty of the `Parts` enum members are omitted from code and comments.